app/services/application_service.rb

```diff
@@ -8,4 +8,24 @@ def call
   def self.call(...)
     new(...).call
   end
+
+  private
+
+  def user_mailer_class(service)
+    case service
+    when :placements
+      Placements::UserMailer
+    when :claims
+      Claims::UserMailer
+    end
+  end
+
+  def support_user_mailer_class(service)
+    case service
+    when :placements
+      Placements::SupportUserMailer
+    when :claims
+      Claims::SupportUserMailer
+    end
+  end
```

Comment:
Where do we think these should live? They're only relevant to services and each method is used in two services. Maybe these could live in the application mailer instead?